### PR TITLE
CyTube Presence

### PR DESCRIPTION
**Pull request by MulverineX**

### CODE REVIEW

File: CyTube/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "name": "MulverineX",
+    "id": "178551656714076161"
+  },
+  "service": "CyTube",
+  "description": {
+    "en": "Content Sharing Platform with wide platform support of simultaneously enjoying videos/music with a live chat and tons of customization/moderation tools"
+  },
+  "url": "cytu.be",
+  "version": "1.0.0",
+  "logo": "https://cdn.discordapp.com/app-assets/653639828826750976/653686085808947240.png?size=512",
```
Comment (Bas950) [inline CyTube/dist/metadata.json]: Upload the image to imgur or something Discords CDN is not allowed.

File: CyTube/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "thumbnail": "https://i.imgur.com/kKZRocg.png",
+  "color": "#e91e63",
+  "tags": [
+    "video",
+    "music",
+    "media"
+  ],
+  "category": "videos",
+  "regExp": "cytu.be",
```
Comment (Alanexei) [inline CyTube/dist/metadata.json]: Don't use regExp unless you are using a regular expression.

File: CyTube/dist/metadata.json (hunk 1 above)
Comment (Alanexei) [inline CyTube/dist/metadata.json]: Please use something like imgur instead. Discord's CDN is not permitted.

**Comment by MulverineX** [comment]
image
image
image
image
image
image

**Pull request merged**